srcpkgs/giara/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'giara'
+pkgname=giara
+version=0.3
+revision=1
+build_style=meson
+hostmakedepends="gobject-introspection python3-setuptools pkg-config glib-devel gettext"
+makedepends="gtk+3-devel libhandy1-devel"
+depends="python3-BeautifulSoup4 python3-cairo python3-dateutil python3-gobject
+ python3-mistune python3-Pillow python3-praw gtksourceview4 webkit2gtk"
```

Comment:
Hmm, another dependant on mistune... Do you know if they have plans to support mistune 2?

Reply:
I don't know, but isn't mistune 2 still in alpha?

Reply:
It is, and I'm no longer planning on packaging it.